### PR TITLE
Add an explicit repositories definitions in ArgoCD CR

### PR DESCRIPTION
**Pull request by jfilipcz**
So we can easily SSL-verify-ignore them:)

### CODE REVIEW

File: docs/1-the-manual-menace/2-argocd.md [hunk 1]
```diff
@@ -76,15 +76,16 @@ When something is seen as not matching the required state in Git, an application
     namespaces:
       - ${TEAM_NAME}-ci-cd
     argocd_cr:
-      repositoryCredentials: |
-        - url: https://${GIT_SERVER}
+      initialRepositories: |
```
Comment (springdo) [inline docs/1-the-manual-menace/2-argocd.md:79]: putting my @pabrahamsson hat on, I believe he'll want to see this whitespace char gone 😉  

But yeah Im happy to merge if no one has any other qualms!

bitmoji

**Review by springdo — CHANGES_REQUESTED** [review]

File: docs/1-the-manual-menace/2-argocd.md (hunk 1 above)
Comment (springdo) [reply docs/1-the-manual-menace/2-argocd.md:79]: An I know im being an absolute pedant with this change req 🍨

Comment (jfilipcz) [reply docs/1-the-manual-menace/2-argocd.md]: That's cool ;) I've removed the extra whitespace

**Review by springdo — APPROVED** [review]

**Comment by springdo** [comment]
BTW - we need you to start putting some emoji on those commit messages.... 🦚

**Pull request merged**